addons/stock/report/report_stock_quantity.py

```diff
@@ -89,15 +88,15 @@ def init(self):
         ELSE date::date - interval '1 day'
     END, '1 day'::interval)::date date,
     CASE
-        WHEN (ls.usage = 'internal' OR ls.usage = 'transit' AND ls.company_id IS NOT NULL) AND ld.usage != 'internal' AND m.state = 'done' THEN product_qty
-        WHEN ls.usage != 'internal' AND (ld.usage = 'internal' OR ld.usage = 'transit' AND ld.company_id IS NOT NULL) AND m.state = 'done' THEN -product_qty
-        WHEN (ls.usage = 'internal' OR ls.usage = 'transit' AND ls.company_id IS NOT NULL) AND ld.usage != 'internal' THEN -product_qty
-        WHEN ls.usage != 'internal' AND (ld.usage = 'internal' OR ld.usage = 'transit' AND ld.company_id IS NOT NULL) THEN product_qty
+        WHEN (whs.id IS NULL AND whd.id IS NOT NULL) OR ld.usage = 'transit' AND m.state = 'done' THEN product_qty
```

Comment:
@sla-subteno-it I don't know if you applied it on some servers but there is a small mistake here. It shoud be whs.id IS NOT NULL AND whd.id IS NULL. Also the opposite bellow

Reply:
> Also I will remove the check on transit. I don't think they remain useful



> Also I will remove the check on transit. I don't think they remain useful

If we will transfer a stock between warehouses, the outgoing and incoming will not see in report stock.

Reply:
> @sla-subteno-it I don't know if you applied it on some servers but there is a small mistake here. It shoud be whs.id IS NOT NULL AND whd.id IS NULL. Also the opposite bellow

Yes, I applied in server. Possible a mistake, I'll check it.

Reply:
> > Also I will remove the check on transit. I don't think they remain useful
> 
> If we will transfer a stock between warehouses, the outgoing and incoming will not see in report stock.

Okay agree
@oma-odoo